tests/components/daikin/test_config_flow.py

```diff
@@ -95,7 +95,7 @@ async def test_discovery(hass, mock_daikin):
 async def test_device_abort(hass, mock_daikin, s_effect, reason):
     """Test device abort."""
     flow = init_config_flow(hass)
-    mock_daikin.side_effect = s_effect
+    mock_daikin.factory.side_effect = s_effect
```

Comment:
@MartinHjelmare I can't get side_effect to work. Do you have any hints on what might be going wrong here?

Reply:
@mlemainque Do you have any smart ways to make this test work?

Reply:
https://dev.azure.com/home-assistant/Core/_build/results?buildId=41240&view=logs&j=67510a0b-4400-5e74-cb95-9572caf5ca54&t=f8fe93fb-5ebb-59cc-455e-c0f2d9ad3a42&l=248
```
s_effect = <class 'concurrent.futures._base.TimeoutError'>
reason = 'device_timeout'

    @pytest.mark.parametrize(
        "s_effect,reason",
        [(asyncio.TimeoutError, "device_timeout"), (Exception, "device_fail")],
    )
    async def test_device_abort(hass, mock_daikin, s_effect, reason):
        """Test device abort."""
        flow = init_config_flow(hass)
        mock_daikin.factory.side_effect = s_effect
    
        result = await flow.async_step_user({CONF_HOST: HOST})
>       assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
E       AssertionError: assert 'create_entry' == 'abort'
E         - abort
E         + create_entry

tests/components/daikin/test_config_flow.py:101: AssertionError
```

Reply:
@MartinHjelmare I think the `.side_effect` doesn't work as we are calling the Appliance a bit different now: https://github.com/home-assistant/core/pull/34807/files#diff-9ba0b5cb03cb9763543902445049c666L38, please advise how we can make these last two tests pass.